packages/components/src/css/component/Icon.module.scss

```
@@ -100,44 +80,13 @@
-
-    background: var(--danger-100);
-    color: var(--danger-900);
-}
-
-.iconBoxedInfo {
+@include mixin.color-variants(iconBoxed) using ($color) {
     composes: iconBoxedColored;
 
-    background: var(--info-100);
-    color: var(--info-900);
+    background: var(--#{$color}-100);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)